### ISSUE TITLE
Can't open PR details in bitbucket

### ISSUE THREAD

**Issue by exdeniz** [TeamCodeStream/codestream#1511]
Can't open PR details in bitbucket. I have full rights to this repository, I am the administrator of it. Everything opens fine on the web, it gives an error through Codestream: 
```
Error Loading Pull Request:
Forbidden: You do not have access to view this workspace.
```

- CodeStream Extension version: v14.22.0
- IDE Version:  VS Code 1.79.0
- OS Version: OSX 13.4

Steps to Reproduce:

1. Open PR List
2. Choose PR and click 'PR Details'
3. Getting an error `Error Loading Pull Request:
Forbidden: You do not have access to view this workspace.`

```
[2023-06-09 07:35:18:699] [221] ScmManager.getRepos completed • 20 ms
[2023-06-09 07:35:18:707] Bitbucket: Completed GET /repositories/work/frontend/pullrequests/71/comments?pagelen=100 • 639 ms
[2023-06-09 07:35:18:806] Bitbucket: Completed GET /repositories/work/frontend/pullrequests/68 • 759 ms
[2023-06-09 07:35:18:824] Bitbucket: Completed GET /repositories/work/frontend/pullrequests/71/activity?pagelen=50 • 756 ms
[2023-06-09 07:35:18:849] Bitbucket: Completed GET /repositories/work/frontend/pullrequests/71/commits • 780 ms
[2023-06-09 07:35:18:911] Bitbucket: Completed GET /repositories/work/frontend/pullrequests/71 • 844 ms
[2023-06-09 07:35:19:260] Bitbucket: Completed GET /user • 578 ms
[Warn  - 10:35:19] [2023-06-09 07:35:19:289] handleErrorResponse:json: {
    "type": "error",
    "error": {
        "message": "You do not have access to view this workspace."
    }
}
Ignoring duplicate error {
  key: 'xxxxxxxxxxxxxxxxxx',
  message: 'Forbidden: You do not have access to view this workspace.'
}
[2023-06-09 07:35:19:289] Bitbucket: FAILED(0x) GET https://api.bitbucket.org/2.0/workspaces/work/members • 608 ms
[Error - 10:35:19] [2023-06-09 07:35:19:289] getPullRequest — {"request":{"providerId":"bitbucket*org","pullRequestId":"{\"id\":71,\"pullRequestId\":71,\"repoWithOwner\":\"work/frontend\"}","src":"getPullRequestConversations","force":true}}
Error: Forbidden: You do not have access to view this workspace.
Error: Forbidden: You do not have access to view this workspace.
    at No.handleErrorResponse (/Users/igorpnev/.vscode/extensions/codestream.codestream-14.22.0/src/providers/thirdPartyProviderBase.ts:542:10)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at No.fetch (/Users/igorpnev/.vscode/extensions/codestream.codestream-14.22.0/src/providers/thirdPartyProviderBase.ts:463:19)
    at async Promise.all (index 5)
    at No.getPullRequest (/Users/igorpnev/.vscode/extensions/codestream.codestream-14.22.0/src/providers/bitbucket.ts:1565:25)
[2023-06-09 07:35:19:290] [222] BitbucketProvider.getPullRequest completed • 609 ms
```

**Comment by planteater** [comment]
@exdeniz - I'm admittedly not as familiar with Bitbucket, but any chance your workspace is limiting oauth access in any way? I know that on GitHub organization admins can do something like that.

**Comment by planteater** [comment]
Fixed in CodeStream 14.24.
